### DISCUSSION TITLE
Document update conflict,multi data provider

### DISCUSSION THREAD

**Discussion by lackray** [Budibase/budibase#5720] [Help] [answered]
when i create multi data provider on one page,it cause the Document update conflict ERROR.
![image](https://user-images.githubusercontent.com/19989329/166874778-226bda4e-203e-405e-b23b-7f46cf71c477.png)

**Comment by lackray** [comment]
It runs fine until recent update.Revert to old version would be help solve this error?@shogunpurple

**Reply by robpegurri** [reply]
We have the same problem, it is serious! I tried to revert to old version using the option in the GUI but it does not work anyway. Does anybody know how to go back to the previous version?

**Comment by shogunpurple** [comment, marked as answer]
@lackray @RobertPegu could you provide some more information? I just added a screen with a bunch of data providers and did not encounter this on both cloud and self hosted environments.

Please use the bug template that we provide for raising bugs and provide some more detail and I will be able to help further. I don't know if you are self hosted, if this is an existing app, what operating system you are on, nothing.

In the meantime, assuming that you are self hosted - you can roll back the version of your budibase installation by updating your  `docker-compose.yaml` to point to the previous version of budibase.

```
  app-service:
    restart: unless-stopped
    image: budibase/apps:v1.0.110
```

Thanks!

**Reply by robpegurri** [reply]
I am self-hosted, I tried exporting my entire app to Budibase Cloud but I still have errors, if you need you could have a look to my app directly. I am opening a bug report with the template right now

**Reply by lackray** [reply]
Didnot expect your reply so soon,thanks.Newbie here,my bad for not use the bug template.The problem dont show any more right row>_<

**Comment by oswaldB** [comment]
Hello I face exactly the same problems with the cloud version.

Try to revert as well but it is not working...

**Reply by oswaldB** [reply]
I mean the bug is still there.
